great_expectations/rule_based_profiler/expectation_configuration_builder/default_expectation_configuration_builder.py

```diff
@@ -14,36 +14,48 @@
 class DefaultExpectationConfigurationBuilder(ExpectationConfigurationBuilder):
     """
     Class which creates ExpectationConfiguration out of a given Expectation type and
-    parameter_name-to-parameter_fully_qualified_parameter_name map (name-value pairs supplied as kwargs dictionary).
+    parameter_name-to-parameter_fully_qualified_parameter_name map (name-value pairs supplied as expectation_kwargs and
+    kwargs dictionaries).
     """
 
-    def __init__(self, expectation_type: str = None, **kwargs):
+    def __init__(
+        self,
+        expectation_type: str,
+        expectation_kwargs: Optional[Dict[str, str]] = None,
```

Comment:
Interesting - so the user can pass in a dict here which will get merged with any kwargs passed in. Is the idea that the user can be more explicit in config using this interface (vs kwargs)?

Reply:
@anthonyburdi It is simpler -- once I realized that all we had to do was to match the signature of the constructor of `ExpectationConfiguration` -- once that is done, everything else falls into place (as long as the Rule's YML follows suite; otherwise, standard errors will be shown, as appropriate).